lib/Tags/Tags.js

```diff
@@ -74,13 +76,13 @@ class Tags extends React.Component {
     const {
       getEntity,
       getEntityTags,
+      entityTagsPath,
     } = this.props;
-    const entity = getEntity(this.props);
+    const entity = cloneDeep(getEntity(this.props));
```

Comment:
Why is `cloneDeep` necessary now if it wasn't before?

Reply:
Well, I used it because `set` modifies an entity, so I decided to work with copy, but I see that the previous approach also mutated the entity, so probably it's not really necessary here, I'll return as it was, thanks.